### DISCUSSION TITLE
Fluentd installation on Linux with custom directory (other than /opt) using rpm or any other mode

### DISCUSSION THREAD

**Discussion by deepaksb2** [ashie/td-agent-builder#476] [Q&A] [answered]
Currently td-agent is installed on Linux (RHEL/CentOS etc.) on default /opt directory using rpm at least, I want to install the fluentd on non /opt directory (due to some restrictions). Is there any way to achieve this?  Or is there any way to change the root path of /opt to any other path post installation (installed using rpm package)? Any help in this regard would be greatly appreciated.

**Comment by kenhys** [comment, marked as answer]
td-agent is intentionally designed to install /opt/td-agent currently, and it's not relocatable.
You can build your custom package by fluent-package-builder, but it is *NOT* recommended and it is *NOT* td-agent (variant of something) obviously, hard to ask your help from the community.
